### PR TITLE
method get_latest_release

### LINKED ISSUE #176: Implement GitLab method for latest release
```python
    def get_latest_release(self) -> GitlabRelease:
        raise NotImplementedError()
```

- https://python-gitlab.readthedocs.io/en/stable/gl_objects/projects.html#project-releases
- https://docs.gitlab.com/ee/api/releases/

### PR DESCRIPTION
**Pull request by lbarcziova**
Fixes #176

### CODE REVIEW

File: ogr/services/gitlab.py [hunk 1]
```diff
@@ -398,7 +398,11 @@ def create_release(
         )
 
     def get_latest_release(self) -> GitlabRelease:
-        raise NotImplementedError()
+        releases = self.gitlab_repo.releases.list()
+        return self._release_from_gitlab_object(
+            raw_release=releases[0],
```
Comment (lachmanfrantisek) [inline ogr/services/gitlab.py]: Are we sure it is the latest?

Comment (lbarcziova) [reply ogr/services/gitlab.py]: https://docs.gitlab.com/ee/api/releases/index.html#list-releases
> Paginated list of Releases, sorted by released_at.

Comment (lachmanfrantisek) [reply ogr/services/gitlab.py]: OK, thanks! Maybe add that note to comment.

Comment (lachmanfrantisek) [reply ogr/services/gitlab.py]: Thanks. I would maybe swap the two lines, but we can leave it as is.

**Comment by lachmanfrantisek** [comment]
Thanks!

**Pull request merged**